Sources/ArcGISToolkit/Components/Offline/PreplannedMapModel.swift

```diff
@@ -322,7 +322,7 @@ extension PreplannedMapArea: PreplannedMapAreaProtocol {
     }
 }
 
-private extension FileManager {
+extension FileManager {
```

Comment:
```suggestion
package extension FileManager {
```
The file URL helpers can be implementation details contained within the package scope